packages/console/src/pages/Connectors/index.tsx

```diff
@@ -112,11 +119,21 @@ const Connectors = () => {
             {
               title: <ConnectorStatusField />,
               dataIndex: 'status',
-              colSpan: 5,
+              colSpan: 4,
               render: (connectorGroup) => <ConnectorStatus connectorGroup={connectorGroup} />,
             },
+            {
+              title: null,
+              dataIndex: 'delete',
+              colSpan: 1,
+              render: (connectorGroup) =>
+                // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+                isConnectorsInTrial ? (
```

Comment:
should we just keep the delete button outside?

Reply:
trial sounds a little bit weird

Reply:
Updated with 'demo'.
In our demo connector, we cannot check its details, so we need to delete them in the list.